uswds/uswds-site
# USWDS-Site - Dependencies: Use node LTS 18

# Summary

Updating to node LTS 18.

## Related issue

Related to uswds/uswds#5010.

## Preview link

[Preview link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-feature-node-18-update/)

## Problem statement

Updating to stay up-to-date with latest Node LTS and dependencies. As well as avoiding compatibility issues with CloudGov pages once node 16 is deprecated.

## Testing and review

1. Run `npm install`.
2. Run `bundle install`.
3. Run `npm start`.
4. Site should run & compile code without errors.
